1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
+Desglosar el proceso en pasos permitirá que cada inmigrante forzado pueda partir desde el paso que se adapte a su situación particular y luego si, continuar con los pasos que restan.
+
+__*¿Quiénes son los clientes potenciales asociados con mi producto?*__
+
+El público al que nos dirigimos son migrantes entre 18 y 45 años que llegan al País Vasco. Nos hemos centrado en esta franja de edad, ya que según una encuesta realizada por el Gobierno Vasco en el año 2010 el 47% de los inmigrantes que llegan tienen entre 25 y 44 años, un 34% son menores de 25 años y tan solo el 19% son mayores de 45 años. Por esta razón, hemos concretado nuestro cliente potencial en este rango de edad, ya que además son las personas que más uso hacen de plataformas web, redes sociales y aplicaciones para móvil o tablets. 
+
+__*¿Cómo puedo ganarme la confianza de estas personas por medio de mi producto?*__
+
+Por medio de las alianzas estratégicas con las ONG´s y la certificación de ellas, ya que desde nuestro sitio web se generan hipervínculos directos a los sitios web de esas organizaciones acreditadas y de igual manera conseguir que esas organizaciones nos mencionen en sus sitios. Por otro lado, los documentos y trámites disponibles en el sitio web deberán cumplir con los siguientes requerimientos: ser verídicos, formales, claros y de sitios oficiales de las instituciones públicas del País Vasco y de reconocidas organizaciones no gubernamentales.
+
+
 ### Circunstancias de uso
 
+__Quien__:
+
+El público al que nos dirigimos son migrantes de entre 20 y 45 años que llegan al País Vasco. Nos hemos centrado en esta franja de edad, ya que según una encuesta realizada por el Gobierno Vasco en el año 2010 el 47% de los inmigrantes que llegan tienen entre 25 y 44 años, un 34% son menores de 25 años y tan solo el 19% son mayores de 45 años. Por esta razón, hemos concretado nuestro publico objetivo en este rango de edad, ya que además son las personas que más uso hacen de plataformas web, redes sociales y aplicaciones para móvil o tablets.
+
+__Que__: 
+
+La plataforma web de Aupa , informará a las personas migrantes que se encuentran en el País Vasco. Es una plataforma creada con el fin de que las personas afectadas encuentren ayudas administrativas, refugio, educación, alimentación, asistencia sanitaria o que puedan incorporarse a organizaciones que defiendan sus derechos, como ACNUR u otras ONGs.
```

Comment:
```suggestion
La plataforma web de Aupa informará a las personas migrantes que se encuentran en el País Vasco. Es una plataforma creada con el fin de que las personas afectadas encuentren ayudas administrativas, refugio, educación, alimentación, asistencia sanitaria o que puedan incorporarse a organizaciones que defiendan sus derechos, como ACNUR u otras ONGs.
```

Si conserváis el paso a paso como hilo conductor para el contenido de vuestro sitio web, éste es un buen lugar en el que recalcarlo y definirlo un poco más.